drivers/SmartThings/matter-switch/src/third-reality/init.lua

```diff
@@ -0,0 +1,410 @@
+  if ib.data.value ~= nil then
+    local hue = math.floor((ib.data.value / 0xFE * 100) + 0.5)
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorControl.hue(hue))
+  end
+end
+
+local function sat_attr_handler(driver, device, ib, response)
+  if ib.data.value ~= nil then
+    local sat = math.floor((ib.data.value / 0xFE * 100) + 0.5)
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorControl.saturation(sat))
+  end
+end
+
+local function temp_attr_handler(driver, device, ib, response)
+  if ib.data.value ~= nil then
+    if (ib.data.value < COLOR_TEMPERATURE_MIRED_MIN or ib.data.value > COLOR_TEMPERATURE_MIRED_MAX) then
+      device.log.warn_with({hub_logs = true}, "Device reported color temperature %d mired outside of supported capability range", ib.data.value)
+      return
+    end
+    local temp = utils.round(CONVERSION_CONSTANT/ib.data.value)
+    local most_recent_temp = device:get_field(MOST_RECENT_TEMP)
+    -- this is to avoid rounding errors from the round-trip conversion of Kelvin to mireds
+    if most_recent_temp ~= nil and
+      most_recent_temp <= utils.round(CONVERSION_CONSTANT/(ib.data.value - 1)) and
+      most_recent_temp >= utils.round(CONVERSION_CONSTANT/(ib.data.value + 1)) then
+        temp = most_recent_temp
+    end
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorTemperature.colorTemperature(temp))
+  end
+end
+
+local color_utils = require "color_utils"
+
+local function x_attr_handler(driver, device, ib, response)
+  local y = device:get_field(RECEIVED_Y)
+  --TODO it is likely that both x and y attributes are in the response (not guaranteed though)
+  -- if they are we can avoid setting fields on the device.
+  if y == nil then
+    device:set_field(RECEIVED_X, ib.data.value)
+  else
+    local x = ib.data.value
+    local h, s, _ = color_utils.safe_xy_to_hsv(x, y)
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorControl.hue(h))
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorControl.saturation(s))
+    device:set_field(RECEIVED_Y, nil)
+  end
+end
+
+local function y_attr_handler(driver, device, ib, response)
+  local x = device:get_field(RECEIVED_X)
+  if x == nil then
+    device:set_field(RECEIVED_Y, ib.data.value)
+  else
+    local y = ib.data.value
+    local h, s, _ = color_utils.safe_xy_to_hsv(x, y)
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorControl.hue(h))
+    device:emit_event_for_endpoint(ib.endpoint_id, capabilities.colorControl.saturation(s))
+    device:set_field(RECEIVED_X, nil)
+  end
+end
+
+--TODO setup configure handler to read this attribute.
+local function color_cap_attr_handler(driver, device, ib, response)
+  if ib.data.value ~= nil then
+    if ib.data.value & 0x1 then
+      device:set_field(HUESAT_SUPPORT, true)
+    end
+  end
+end
+
+local third_reality_handler = {
+  NAME = "Third Reality Handler",
+  lifecycle_handlers = {
```

Comment:
You can remove all the `lifecycle_handlers` so here. When these are removed, the device will use the parent driver (`matter-switch`) handlers, which is the desired behavior.